src/integrationTests/euidSdk.test.ts

```diff
@@ -26,12 +25,19 @@ jest.spyOn(document, 'URL', 'get').mockImplementation(() => mockUrl);
 
 const makeIdentity = mocks.makeIdentityV2;
 
+const euidProductDetails: ProductDetails = {
+  name: 'EUID',
+  defaultBaseUrl: 'https://prod.euid.eu',
+  localStorageKey: 'EUID-sdk-identity',
+  cookieName: '__euid',
+};
+
```

Comment:
We could possibly import these product details from `euidSdk.ts` and same for the uid2 product details from `uid2Sdk.ts` instead of redefining them in multiple places - but the point is that `productDetails` is private and if we exported them to import in these test files, would that make them public now?